classes/admin.class.php

```diff
@@ -448,4 +453,26 @@ public function load_admin_menu() {
 		include_once PPOM_PATH . '/classes/survey.class.php';
 		PPOM_Survey::get_instance()->init();
 	}
+
+	/**
+	 * Get the license category (Essential, Plus, VIP).
+	 * 
+	 * @param number Plan ID.
+	 * @return number The associated category.
+	 */
+	public static function get_license_category( $license_plan ) {
+		if ( in_array( $license_plan, array( 1, 4, 9 ) ) ) {
+			return self::LICENSE_PLAN_1;
+		}
+
+		if ( in_array( $license_plan, array( 2, 5, 8 ) ) ) {
+			return self::LICENSE_PLAN_2;
+		}
+
+		if ( in_array( $license_plan, array( 3, 6, 7, 10 ) ) ) {
+			return self::LICENSE_PLAN_3;
+		}
```

Comment:
You can group the categories in an array so one can manage it from one place: https://github.com/Codeinwp/store-themeisle/blob/e3809118f99afd1f074daabdf03963b27297d6a4/web/app/mu-plugins/edd-sl-booster/src/Utils/Download.php#L266